xarray/tests/test_backends.py

```diff
@@ -1179,9 +1191,9 @@ def test_encoding_unlimited_dims(self):
 
 
 # tests pending h5netcdf fix
-@pytest.mark.xfail
-class H5NetCDFDataTestAutocloseTrue(H5NetCDFDataTest):
-    autoclose = True
+# @pytest.mark.xfail  @shoyer - is this still an issue?
```

Comment:
Yes, I'm afraid so. It's probably an h5py issue -- not really clear what's going on.

Reply:
reminder to myself to remove this comment